packages/did/src/DidLinks/AccountLinks.chain.ts

```diff
@@ -77,75 +66,22 @@ interface PalletDidLookupLinkableAccountLinkableAccountId extends Enum {
   readonly type: 'AccountId20' | 'AccountId32'
 }
 
-type AssociateAccountRequestValue = [
-  string | Uint8Array, // AccountId
-  string | Uint8Array | EncodedSignature // signature
-]
-
-/**
- * Type required for encoding Enum type for association request extrinsics.
- */
-type AssociateAccountRequest =
-  | { Dotsama: AssociateAccountRequestValue }
-  | { Ethereum: AssociateAccountRequestValue }
-
-/**
- * Api augmentation override for when the ethereum enabled pallet version has landed.
- */
-type WithEtherumSupport = {
-  tx: {
-    didLookup: {
-      associateAccount: AugmentedSubmittable<
-        (
-          req: AssociateAccountRequest,
-          expiration: u64 | AnyNumber | Uint8Array
-        ) => SubmittableExtrinsic
-      >
-      removeAccountAssociation: AugmentedSubmittable<
-        (account: EncodedMultiAddress) => SubmittableExtrinsic,
-        [PalletDidLookupLinkableAccountLinkableAccountId]
-      >
-    }
-  }
-  query: {
-    didLookup: {
-      connectedDids: AugmentedQuery<
-        'promise',
-        (arg: EncodedMultiAddress) => Option<PalletDidLookupConnectionRecord>,
-        [PalletDidLookupLinkableAccountLinkableAccountId]
-      >
-      connectedAccounts: AugmentedQueryDoubleMap<
-        'promise',
-        (
-          didId: string | Uint8Array,
-          accountId: EncodedMultiAddress
-        ) => Option<bool>,
-        [AccountId32, PalletDidLookupLinkableAccountLinkableAccountId]
-      >
-    }
-  }
-}
-
 /**
  * Detects whether api augmentation indicates presence of Ethereum linking enabled pallet.
```

Comment:
oh technically this should say decoration

Reply:
Updated in https://github.com/KILTprotocol/sdk-js/pull/613/commits/382c070784cf24911516f96e268166847e0bc5fe